src/pipes/input-characters-validation.ts

```diff
@@ -0,0 +1,40 @@
+import { BadRequestException, Injectable, InternalServerErrorException, PipeTransform } from '@nestjs/common';
+import { ACBS } from '@ukef/constants';
+
+@Injectable()
+export class InputCharacterValidationPipe implements PipeTransform {
+  transform(value: any, metadata) {
```

Comment:
Can we type `metadata` as `metadata: ArgumentMetadata` please, and type the return value?